### DISCUSSION TITLE
One device listed two times in HA

### DISCUSSION THREAD

**Discussion by AlexK98** [softwarecrash/Solar2MQTT-ESP8266#189] [Q&A]
Hello,

Is it normal that in HA I see two items for one physical device?

![2024-12-07 23 52 21](https://github.com/user-attachments/assets/9712fa93-f08c-47c3-9f2c-9d1336e1b19f)

I thought it is some bug and deleted it, but it reappears.
Besides, the lower one (marked with red) shows sensors that I disabled in the main (upper) item. Is this intentional?

**Comment by softwarecrash** [comment]
nope, thats not normal
what firmware version?
debug data from HA?

**Reply by AlexK98** [reply]
Firmware 2.0.0-Pre6. I will experiment a bit more cause I deleted both entries and now only 1 was auto added.

**Reply by softwarecrash** [reply]
> Firmware 2.0.0-Pre6. I will experiment a bit more cause I deleted both entries and now only 1 was auto added.

this can come from some changes in the firmware and renaiming. when now only one device is listed its okay

**Reply by AlexK98** [reply]
> this can come from some changes in the firmware and renaiming. when now only one device is listed its okay

With the latest firmware Pre6A3 only one device is being listed. Maybe listing 2 devices was a glitch of HA-Mosquito.
